app/controllers/admin/auctions_controller.rb

```diff
@@ -3,7 +3,7 @@ class AuctionsController < ApplicationController
     before_filter :require_admin
 
     def index
-      @auctions = Auction.all.map { |auction| Presenter::AdminAuction.new(auction) }
+      @auctions = Auction.all.map { |auction| AdminAuctionPresenter.new(auction) }
```

Comment:
No I think that is an admirable goal and I have no issue with that, but I just want to make sure @adelevie is aware of this